apps/sms/test/marionette/drafts_test.js

```diff
@@ -283,6 +289,34 @@ marionette('Messages Drafts', function() {
       });
     });
 
+    suite('Thread-less recipient-less drafts', function() {
```

Comment:
> In Inbox "conversation-less draft" is an item of the list, don't know if we should call this a "conversation" but at least it has the same info :) maybe in Inbox we should use the more general term "inbox item" :)

Heh, note that we're going to return both conversations and conversation-less drafts as "conversation/conversation-info" via ```ConversationService.getAllConversations``` :) [1]

[1] https://github.com/mozilla-b2g/gaia/blob/master/apps/sms/services/README.md